src/Symfony/Component/Translation/TranslatorBagInterface.php

```diff
@@ -24,8 +24,7 @@ interface TranslatorBagInterface
      * Gets the catalogue by locale.
      *
      * @param string|null $locale The locale or null to use the default
-     *
-     * @return MessageCatalogueInterface
+     * @return MessageCatalogueInterface|null
```

Comment:
for a lower branch?

Reply:
I had to fix it due to psalm report. But yes, it could be fixed for a lower branch

Reply:
Done here https://github.com/symfony/symfony/pull/40405